<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge Detection Methods</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>470</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxEdgeDetection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Edge Detection</string>
   </property>
   <widget class="QRadioButton" name="radioButtonCanny">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSobelX">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSobelY">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>160</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLaplacian">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>230</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplacian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSobelXY">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>180</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel XY</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCannyHighThreshold">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HIGH Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCannyLowThreshold">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>76</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCannyKernel">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>105</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order of Kernel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCannyHighThreshold">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>47</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCannyLowThreshold">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>74</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCannyKernel">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSobelKernel">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>205</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSobelKernel">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>203</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSIFT">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>250</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonORB">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>270</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ORB</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOrbMaxFeatures">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>290</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxKMeans">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Means</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDominantColors">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>370</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
